pkgs/development/python-modules/country-list/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "country_list";
+  version = "1.1.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "bulv1ne";
+    repo = "country_list";
+    tag = "v${version}";
+    hash = "sha256-jG2AC8c6mgWjHVBX7XW021PLPliLTwEBkN6+HSecfL4=";
+    fetchSubmodules = true;
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  meta = with lib; {
+    description = "List of all countries with names and ISO 3166-1 codes in all languages.";
```

```suggestion
    description = "List of all countries with names and ISO 3166-1 codes in all languages";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes